mappings/net/minecraft/client/gui/screen/ingame/SmithingScreen.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_4895 net/minecraft/client/gui/screen/ingame/SmithingScreen
-	FIELD field_22795 TEXTURE Lnet/minecraft/class_2960;
+	FIELD field_22795 BACKGROUND Lnet/minecraft/class_2960;
```

Comment:
Other screens use `TEXTURE`.

Reply:
Other screens should be changed, then. This texture is used to draw the background of the screen.

Reply:
hmm, i suggest moving that to a separate pr